### PR TITLE
update rc.local and add run.sh

### PR DESCRIPTION
**Pull request by Trotyl15**

### CODE REVIEW

File: imaging/rc.local.copyme [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
-# Copy 1 of the following contents to the end of rc.local:
+# Copy the following contents to the end of rc.local:
 
-# OpenCV Python
-touch /home/warg/record.log  # To confirm script has run
-cd /home/warg/model-training/
-source venv/bin/activate
-cd imaging/
-python -m imaging
+/home/warg-rpi4b/model-training/imaging/run.sh
```
Comment (Xierumeng) [inline imaging/rc.local.copyme]: `/home/warg/...`

File: imaging/rc.local.copyme [hunk 2]
```diff
@@ -1,14 +1,6 @@
...
 
-# FFMPEG video
-touch /home/warg/record.log  # To confirm script has run
-cd /home/warg/model-training/imaging/
-./video.sh
+exit 0
```
Comment (Xierumeng) [inline imaging/rc.local.copyme]: Is this necessary?

File: imaging/run.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Colour calibration 2023-10-12
+v4l2-ctl -d /dev/video0 --set-ctrl hue=-128
+v4l2-ctl -d /dev/video0 --set-ctrl white_balance_automatic=0
+v4l2-ctl -d /dev/video0 --set-ctrl white_balance_temperature=6500
+v4l2-ctl -d /dev/video0 --set-ctrl auto_exposure=1
+
+# Comment out one of the following
```
Comment (Xierumeng) [inline imaging/run.sh]: Remove this, no longer using FFMPEG.

File: imaging/run.sh [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+touch /home/warg/record.log  # To confirm script has run
+cd /home/warg/model-training/
+source venv/bin/activate
+cd imaging/
+python -m imaging
+
+# FFMPEG video
+# touch /home/warg/record.log  # To confirm script has run
+# cd /home/warg/model-training/imaging/
+# ./video.sh
```
Comment (Xierumeng) [inline imaging/run.sh]: Remove this, no longer using FFMPEG.

**Review by Xierumeng — COMMENTED** [review]
Reviewed.

File: imaging/rc.local.copyme (hunk 1 above)
Comment (Trotyl15) [reply imaging/rc.local.copyme:4]: Just curious, is there a specific reason we want to use `warg`? Is it for generality?

File: imaging/rc.local.copyme [hunk 5]
```diff
@@ -1,6 +1,4 @@
 #!/bin/bash
 # Copy the following contents to the end of rc.local:
 
-/home/warg-rpi4b/model-training/imaging/run.sh
-
-exit 0
\ No newline at end of file
+/home/warg/model-training/imaging/run.sh
```
Comment (Xierumeng) [inline imaging/rc.local.copyme]: Add empty line at end of file.

File: imaging/run.sh [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+v4l2-ctl -d /dev/video0 --set-ctrl white_balance_automatic=0
+v4l2-ctl -d /dev/video0 --set-ctrl white_balance_temperature=6500
+v4l2-ctl -d /dev/video0 --set-ctrl auto_exposure=1
+
+# OpenCV Python
+touch /home/warg/record.log  # To confirm script has run
+cd /home/warg/model-training/
+source venv/bin/activate
+cd imaging/
+python -m imaging
```
Comment (Xierumeng) [inline imaging/run.sh]: Empty line at end of file.

**Review by Xierumeng — COMMENTED** [review]
Reviewed.

**Review by Xierumeng — APPROVED** [review]
Approved.

**Pull request merged**